index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <!-- <link rel="stylesheet" href="style.css" type="text/css"> -->
+    <link rel="stylesheet" href="style1.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@300;400;500;600;700&display=swap" rel="stylesheet">
+    <title>Portfolio</title>
+</head>
+<body>
+<section>
+    <header>
+        <nav class="nav-bar">
+            <a href="#" class="nav-title">Welcome</a>
+            <a href="#" class="hamburger"><img src="Images/Enabled.svg" alt="hamburger"></a>
+        </nav>
+    </header>
+    <div class="hero">
+        <div class="hero-container">
```

Comment:
- [ ] consider changing this div tag into a section tag to fulfill this requirement 👇 
> The container of the section is a `section` tag